### PR TITLE
fix(ingestion/vertica): support columns with timestamp precision

### LINKED ISSUE #5295: Vertica timestamp(0) causes an unable to get column information error in DataHub CLI
**Describe the bug**
A warning is received in Datahub CLI while executing ingestion from Vertica DB via Vertica plugin.
`Warning output: 'warnings': {'XXX_TABLE': ['unable to get column information due to an error -> __init__() got an unexpected keyword ' "argument 'precision'"]}`

It seems that the error is caused when an attribute data type is set to timestamp(0). The error can be bypassed when commenting out the following lines in venv\Lib\site-packages\datahub\ingestion\source\sql\vertica.py, `_get_column_info` function.
```
elif attype in ("timestamp", "time"):
        kwargs["timezone"] = False
        # if charlen:
        #     kwargs["precision"] = int(charlen)
        args = ()
```
The created kwargs object precision attribute isn't compatible with the timestamp parameters: `{'timezone': False, 'precision': 0}`, it's possible that the same error would appear when working with timestampz data type.

Commenting most likely works as timestamp precision is optional -[ TIMESTAMP/TIMESTAMPTZ](https://www.vertica.com/docs/9.3.x/HTML/Content/Authoring/SQLReferenceManual/DataTypes/Date-Time/TIMESTAMP.htm)

**To Reproduce**
Create a table in Vertica including an attribute with timestamp(0):
```
CREATE TABLE YYY.XXX_TABLE
(
    load_timestamp timestamp(0)
);
```
Execute the pipeline using datahub CLI.

**Expected behavior**
Load meta from Vertica to file using datahub CLI w/o warnings.

**Desktop (please complete the following information):**
 - Python 3.8.2
 - acryl-datahub 0.8.40 / DataHub CLI 0.8.40
 - wheel 0.37.1
 - setuptools 62.6.0
 - pip 22.1.2

### PR DESCRIPTION
**Pull request by inancdokurel**
Fixes #5295

It is not a perfect fix, however it allows timestamp fields with precision values to be ingested into Datahub 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by inancdokurel** [comment]
Anybody willing to review and provide comments?

File: metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py [hunk 1]
```diff
@@ -175,6 +207,7 @@ def _get_column_info(  # noqa: C901
             break
 
     self.ischema_names["UUID"] = UUID
+    self.ischema_names["TIMESTAMP"] = TIMESTAMP_WITH_PRECISION
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py:210]:
```suggestion
    self.ischema_names["TIMESTAMP"] = TIMESTAMPWP
```

File: metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py [hunk 2]
```diff
@@ -31,9 +31,41 @@ class UUID(String):
...
+         ignored.
+        :param precision: integer.  Indicates the PRECISION field when provided
+
+
+        """
+        super(TIMESTAMP, self).__init__(timezone=timezone)
+        self.precision = precision
+
+
+def TIMESTAMP_WITH_PRECISION(*args, **kwargs):
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py:62]: this method isn't necessary - see below

**Review by hsheth2 — CHANGES_REQUESTED** [review]
Sorry about the delay in review

This mostly looks good, but I've added two small comments

File: metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py (hunk 1 above)
Comment (inancdokurel) [reply metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py:206]: changed in the latest commit

**Pull request merged**